developer/help/placing-comment-based-help-in-functions.md

```diff
@@ -1,72 +1,77 @@
-﻿---
-title: "Placing Comment-Based Help in Functions | Microsoft Docs"
-ms.custom: ""
-ms.date: "09/12/2016"
-ms.reviewer: ""
-ms.suite: ""
-ms.tgt_pltfrm: ""
-ms.topic: "article"
-ms.assetid: 5ec7159e-e4e9-4b21-95df-94244432f679
-caps.latest.revision: 5
----
-# Placing Comment-Based Help in Functions
-This topic explains where to place comment-based help for a function so that the Get-Help cmdlet associates the comment-based help topic with the correct function.
+---
+title: "Placing Comment-Based Help in Functions | Microsoft Docs"
+ms.custom: ""
+ms.date: "09/12/2016"
+ms.reviewer: ""
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+ms.assetid: 5ec7159e-e4e9-4b21-95df-94244432f679
+caps.latest.revision: 5
+---
+# Placing Comment-Based Help in Functions
+
+This topic explains where to place comment-based help for a function so that the `Get-Help` cmdlet associates the comment-based help topic with the correct function.
+
+## Where to Place Comment-Based Help for a Function
+
+- At the beginning of the function body.
+
+- At the end of the function body.
+
+- Before the `Function` keyword. When the function is in a script or script module, there cannot be more than one blank line between the last line of the comment-based help and the `Function` keyword. Otherwise, `Get-Help` associates the help with the script, not with the function.
+
+## Examples of Help Placement in a Function
+
+ The following examples show each of the three placement options for comment-based help for a function.
+
+### Help at the Beginning of a Function Body
+
+ The following example shows comment-based at the beginning of a function body.
+
+```powershell
+
+function MyProcess
+{
+    <#
+       .Description
+       The MyProcess function gets the Windows PowerShell process.
+    #>
+
+    Get-Process powershell
+}
+
+```
+
+### Help at the End of a Function Body
+
+ The following example shows comment-based at the end of a function body.
+
+```powershell
+
+function MyFunction
+{
+    Get-Process powershell
+
+    <#
+       .Description
+       The MyProcess function gets the Windows PowerShell process.
+    #>
+}
+
+```
+
+### Help Before the Function Keyword
+
+ The following examples shows comment-based on the line before the function keyword.
+
+```powershell
+
+<#
+    .Description
+    The MyProcess function gets the Windows PowerShell process.
+#>
+function MyFunction { Get-Process powershell}
+
+```


```

Comment:
Control character line ending

Reply:
This was an issue with my code fence logic and is fixed